frontend/app/blueprints/sample/templates/sample.html

```diff
@@ -298,102 +304,157 @@ <h4 id="sample-id-heading" class="card-title">{{sample.sample_id}}</h4>
                         {% endif %}
-                    {% endfor %}
                     </div>
-                    <h5>Quality</h5>
-                    <hr class="mt-2 col-md-3 border border-success border-2">
-                    <div id="qc-heading" class="row gx-5 justify-content-md-center">
-                        {% for qc_tool in sample.qc %}
-                            <div class="col-md-auto">
-                                {% if qc_tool.software == "postalignqc" %}
-                                    {{ qc_card_postalign(qc_tool) }}
-                                {% else %}
-                                    {{ qc_card_generic(qc_tool) }}
-                                {% endif %}
-                            </div>
-                        {% endfor %}
+                {% endfor %}
 
-                        {% for res in sample.typing_result %}
-                            {% if res.type == "cgmlst" %}
-                                <div class="col-md-auto">
-                                {{ cgmlst_qc_card(res) }}
-                                </div>
-                            {% endif %}
-                        {% endfor %}
-                    </div>
-                    <h5 class="mb-0">Analysis metadata</h5>
-                    <hr class="mt-2 col-md-3 border border-success border-2">
-                    <div class="row">
-                        <div class="col-auto">{{ analysis_meta_pipeline(sample) }}</div>
-                        <div class="col-auto">{{ analysis_meta_databases(sample) }}</div>
+                {% for res in sample.typing_result %}
+                    {% if res.type == "cgmlst" %}
+                        <div class="col-md-auto">
+                        {{ cgmlst_qc_card(res) }}
+                        </div>
+                    {% endif %}
+                {% endfor %}
+            </div>
+            <h5 class="mb-0">Analysis metadata</h5>
+            <hr class="mt-2 col-md-3 border border-success border-2">
+            <div class="row">
+                <div class="col-auto">{{ analysis_meta_pipeline(sample) }}</div>
+                <div class="col-auto">{{ analysis_meta_databases(sample) }}</div>
                 </hr>
             </div>
         </div>
     </div>
 </div>
+<script>
+    const apiURL = "{{ config.BONSAI_API_URL }}"
+    const sampleId = document.getElementById("sample-id-heading").textContent 
+
+    function drawDendrogram(newick, sampleId) {
+        const tree = new TidyTree(newick, { 
+            parent: "#tree-body",
+            //layout: "circle",
+            layout: "vertical",
+            type: "dendrogram",
+            mode: "square",
+            ruler: false,
+            leafLabels: true,
+            margin: [10, 10, 70, 30]
+        });
+        // hightlight current sample
+        tree.search(d => d.data.id.includes(sampleId)).selectAll("circle").style("fill", "steelblue").attr("r", 5);
```

Comment:
I would move the "5" into a centralized styling location (which could be separate for the tree from the general CSS). Then you don't need to find your way to `sample.html` when you just want to change the radius later